queries/rescript/highlights.scm

```diff
@@ -14,23 +14,43 @@
   (list_pattern)
 ] @type
 
+((type_identifier) @type.builtin
+  (#any-of? @type.builtin
+    "int" "char" "string" "float" "bool" "unit"))
+
+
+((unit) @constant.builtin
+  (#set! "priority" 105))
+
 [
   (variant_identifier)
   (polyvar_identifier)
-] @constant
+] @constructor
 
 (record_type_field (property_identifier) @property)
 (record_field (property_identifier) @property)
 (object (field (property_identifier) @property))
 (object_type (field (property_identifier) @property))
-(member_expression (property_identifier) @property)
-(module_identifier) @namespace
+(module_identifier) @module
+
+(member_expression (property_identifier) @variable.member)
```

Comment:
This was `@property` and was changed to `@variable.member`. I looked at other languages and this is how it's usually handled: `@property` is for places such as record definition when both the property and value are explicitly written, and member is for extracting a value out of that property. (which is why it belongs to the `variable` group.)